lib/actions/environment-variables.coffee

```diff
@@ -44,15 +44,18 @@ exports.list =
 				throw new Error('You must specify an application or device')
 
 		.tap (environmentVariables) ->
-			if not options.verbose
-				isSystemVariable = resin.models.environmentVariables.isSystemVariable
-				environmentVariables = _.reject(environmentVariables, isSystemVariable)
-
-			console.log visuals.table.horizontal environmentVariables, [
-				'id'
-				'name'
-				'value'
-			]
+			if environmentVariables.length > 0
```

Comment:
A better way to formulate this is to check if the environment variables array is empty, and if so, throw an error right away. This also reduces indentation from the whole table printing block, and removes the need of an `else` block.